server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+const port = 4200
+const path = require('path');
+const bodyParser=require('body-parser');
+const db =require('./database')
+
+app.use(express.static(path.join(__dirname, './client/dist')));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get('/', (req, res) => {
```

Comment:
There is no error handling here as well.  everything here is occurring synchronously which will create race conditions and could break your service later.